docs/examples/trees_example.py

```diff
@@ -0,0 +1,232 @@
+import trees
+from fury import window, actor, lib
+import numpy as np
+
+
+# TESTING A QUADTREE OF RANDOM 2D POINTS
+br = trees.Branch2d(4, 0.0, 1.0, 0.0, 1.0)
```

Comment:
We may need to add a functionality in which the users do not need to deal with branches or add points, etc. You can keep this functionality as advanced usage. But the common usage would be just 
Tree2D(points)
tree.addPoints()
maybe something to delete points as well or to update points with a new array.